app/services/clients/utils/ledger_legacy_connector_parity.js

```diff
@@ -40,6 +40,9 @@ const legacyConnectorTransactionParity = (transaction) => {
 }
 
 const legacyConnectorTransactionSummaryParity = (transactionSummaryResult) => {
+  if (transactionSummaryResult === 'undefined') {
```

Comment:
This will test if the returned value is the exact string 'undefined', if this is testing if this doesn't exist it would need to test the type of the object. 

In this case a falsy comparison would probably work (as it expects an object otherwise, an object will always be truthy)
```js 
if (!transactionSummaryResult) { 
  return {}
}
```

Or to follow the current pattern
```js
if (typeof transactionSummaryResult === 'undefined') { 
  return {}
}
```

Reply:
👍